src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -1006,6 +1015,18 @@ class Diagram extends Component<Props> {
               }}
             />
           )}
+          {formType === 'entity' && (
+            <VariableTypesForm
+              contentLang={this.state.currentLang}
+              customKey={data.id}
+              formData={editingNodeItem}
+              close={() => {
+                this.timeout = setTimeout(() => {
+                  this.setState({ editingNodeItem: null })
+                }, 200)
```

Comment:
why?

Reply:
To keep the node open when switching from one enum to another, and to make the switch seamless. All other panels are usign the same logic